src/Components/Admin/common/deleteModal.js

```diff
@@ -16,16 +16,28 @@ const useStyles = makeStyles(() => ({
     },
 }));
 
+function NewlineText(string) {
+    return string.split("\n").map((str, index) => (
+        <p style={{ fontSize: 14, lineHeight: "16px" }} key={index}>
+            {str}
+        </p>
+    ));
+}
```

Comment:
What's this for?

Reply:
It's so that we can include `\n` characters in the single prop field for the popup and it will display in different paragraphs. It is being used in the delete scene popup (the extra information about deleting transitions is in a new paragraph). It looks better imo. Generalized the logic so other popups can use this too if needed.